src/epics/chatEpics.js

```diff
@@ -53,6 +56,22 @@ import 'rxjs/add/observable/from';
 import 'rxjs/add/observable/fromEvent';
 import 'rxjs/add/observable/merge';
 import 'rxjs/add/observable/if';
+import 'rxjs/add/operator/takeUntil';
+
+const messageEpic = (action$) =>
+  action$.ofType('CHAT_START_SUGGESTIONS')
+    .mergeMap((cursor) => {
+      return action$.ofType('CHAT_MESSAGE_UPDATE')
+      .map((msg) => showSuggestions(cursor.cursorIndex, msg.message))
+      .takeUntil(action$.ofType('CHAT_STOP_SUGGESTIONS'))
+      .catch((err) => console.error(err))
+    })
+
+const suggestionEpic = (action$) =>
+  action$.ofType('CHAT_ADD_SUGGESTION')
+    .map(() => stopSuggestions())
+    .catch((err) => console.error(err))
```

Comment:
@Spetastian Do we have a global error helper we can use for this sort of situation?

Reply:
No, not right now. Also this will break if this observable emits an error. It should always map to an action that redux handles. IMO, we should always tell the user if something goes wrong. 

The lest we should do right now is logging the error to the console and display an alert that something went wrong and that more information can be found in the console.

Reply:
:+1: sounds good